setup.py

```diff
@@ -21,7 +21,7 @@
     description="Asynchronous library to control Shelly devices.",
     long_description=README_FILE.read_text(encoding="utf-8"),
     long_description_content_type="text/markdown",
-    packages=find_packages(),
+    packages=find_packages(exclude=["tests", "tests.*"]),
```

Comment:
```suggestion
    packages=["aioshelly"],
```

Reply:
`find_packages` import should also be removed. @onkelbeh Will you take care of this or should I commit this change?